emailsender/pkg/email/ratelimiter.go

```diff
@@ -0,0 +1,57 @@
+package email
+
+import (
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/stackrox/acs-fleet-manager/emailsender/pkg/db"
+	"time"
+)
+
+const (
+	windowSizeHours = 24
+)
+
+// RateLimiter defines an exact methods for rate limiter
+type RateLimiter interface {
+	Allow(tenantID string) bool
+}
+
+// RateLimiterService contains configuration and dependency for rate limiter
+type RateLimiterService struct {
+	limitPerTenant int
+	dbConnection   db.DatabaseClient
+}
+
+// NewRateLimiterService creates a new instance of RateLimiterService
+func NewRateLimiterService(dbConnection *db.DatabaseConnection) *RateLimiterService {
+	return &RateLimiterService{
+		dbConnection: dbConnection,
+	}
+}
+
+// Allow checks whether specified tenant can send an email for current timestamp
+func (r *RateLimiterService) Allow(tenantID string) bool {
```

Comment:
Nit: because I think Allow does not communicate the intend of using that as a check.
```suggestion
func (r *RateLimiterService) IsAllowed(tenantID string) bool {
```